namespaceFinder.php

```diff
@@ -11,20 +11,35 @@ class namespaceFinder {
     private $wantedNs;
     private $isSafe;
 
+    /**
+     * Resolves the namespace on construction
+     * 
+     * @param string $path the namespace link
+     */
     function __construct($path){
         $this->wantedNs = $this->computeWantedNs($path);
         $this->sanitizeNs();
     }
 
     private function computeWantedNs($path){
         global $ID;
+        global $conf;
+        
+        // Convert all other separators to colons
+        if ($conf['useslash']) $path = str_replace('/', ':', $path);
+        $path = str_replace(';', ':', $path);
```

Comment:
I'm not sure I understand that line: in which case could it make sense to have a `;` in `$path`?

Reply:
`;` as alternate namespace seperator?
See: https://github.com/splitbrain/dokuwiki/issues/3857#issuecomment-1377841463